src/runtime_action.rs

```diff
@@ -15,11 +18,60 @@ pub enum RuntimeAction {
     RateLimit(RateLimitAction),
 }
 
+pub(super) mod errors {
+    use cel_parser::ParseError;
+    use std::fmt::{Debug, Display, Formatter};
+
+    #[derive(Debug)]
+    pub enum NewActionError {
```

Comment:
I know this error is meant specifically for the creation of actions, but what would you say to rename it to `RuntimeActionError` or simply `ActionError`, the former will tell a better story for the actual mod that is issuing the error and the later would be open for a wider scope.

Reply:
fwiw I _think_ there should not be an overlap between other types of errors because the create errors happen on configuration time whereas other errors relating to runtime actions happen at runtime, so I can't imagine a case where you'd need an enum for a function that can output both

Reply:
+1 to renaming it `CreateActionError`, `ActionCreationError` or `ActionIntializationError` or something along those lines. `NewActionError` sounds more like an "order", a verb, than a noun. My $.02

Reply:
Gone with `ActionCreationError`, thanks for the suggestions